lib/cadet/assessments/answer_types/voting_answer.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Cadet.Assessments.AnswerTypes.VotingAnswer do
```

Comment:
What is this entity for? If the submission votes are stored in a separate table already, then this should not be stored in the database. You can generate the "answer" from the SubmissionVotes table and then inject it into the assessment answer field as needed (both for grading and user endpoints).

Reply:
Added VotingAnswer to award students xp and grade when the voting assessment is submitted.